### ISSUE TITLE
Failing accessibility testing software (cypress-axe & wave)

### ISSUE THREAD

**Issue by louis-yung-naimuri** [react-pdf-viewer/react-pdf-viewer#1448]
My team uses cypress-axe and wave for testing the accessibility of our software, we are having an issue were the pdf viewer is failing these tests for these issues:

cypress-axe failures:
- On the popup buttons “ARIA attribute is not allowed: aria-expanded="false"" is happening due to the divs not having a role of some form of input.
- Most of the button are failing due to "Element has focusable descendants", this is caused by divs with roles that allow them to be focused while also having focusable buttons within them
- The scrollable region of the viewer fails "Element should have focusable content Element should be focusable", this happens when there is nothing to focus on in the document. This is also an accessibility issue as if there is nothing to focus on you can't scroll through the file if you are a keyboard user, this is not an issue on firefox but is on chrome.

wave failures:

- Any button with a tool tip is failing  for "Broken ARIA reference for tool tips" as the tooltip does not exist in the DOM until hovered but the reference to it is on the object.
